WabiSabiMonitor.ApplicationCore/AnalysisStoreManager.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Newtonsoft.Json;
+using WabiSabiMonitor.ApplicationCore.Interfaces;
+using WabiSabiMonitor.ApplicationCore.Utils.Bases;
+using WabiSabiMonitor.ApplicationCore.Utils.Helpers;
+using WabiSabiMonitor.ApplicationCore.Utils.WabiSabi.Models.Serialization;
+
+namespace WabiSabiMonitor.ApplicationCore
+{
+    public class AnalysisStoreManager : PeriodicRunner
+    {
+        private readonly IAnalyzer _analyzer;
+        private readonly IRoundsDataFilter _roundsDataFilter;
+
+        public AnalysisStoreManager(IAnalyzer analyzer, IRoundsDataFilter roundsDataFilter, TimeSpan period) : base(period)
+        {
+            _analyzer = analyzer;
+            _roundsDataFilter = roundsDataFilter;
+        }
+
+        protected override async Task ActionAsync(CancellationToken cancel)
+        {
+            var date = DateTime.Now.Date;
+            var path = Path.Combine(EnvironmentHelpers.GetDataDir(Path.Combine("WabiSabiMonitor", "Client")), $"Analysis_{date:yyyy-MM-dd}.json");
```

Comment:
Add a specific subfolder whether inside `Client` whether one level up please because this folder contains Config, it should not be flooded.

If in the precedent PR to split the data in days you didn't use a specific subfolder, please add one as well. You can do this on this PR